modelina-website/src/helpers/GeneratorCode/TypeScriptGenerator.ts

```diff
@@ -20,6 +20,10 @@ export function getTypeScriptGeneratorCode(
     }`);
   }
 
+  if (generatorOptions.tsEnumType) {
+    optionString.push(`enumType: '${generatorOptions.tsEnumType}'`);
```

Comment:
Lets make the generator code a bit prettier 😄 
```suggestion
    optionString.push(`    enumType: '${generatorOptions.tsEnumType}'`);
```

Reply:
Done